website/src/components/pageloads_table.js

```diff
@@ -14,23 +14,26 @@ export const PageloadsTable = ({ encodedStartDate, encodedEndDate }) => {
       })
   }, [encodedStartDate, encodedEndDate])
 
+  console.log(visitsPerCompany)
```

Comment:
At some point we need to clean up the console.log invocations littering this codebase